src/theory/bv/theory_bv_utils.cpp

```diff
@@ -136,35 +127,28 @@ bool isBvConstTerm(TNode node)
 
 bool isBVPredicate(TNode node)
 {
-  if (node.getKind() == kind::EQUAL || node.getKind() == kind::BITVECTOR_ULT
-      || node.getKind() == kind::BITVECTOR_SLT
-      || node.getKind() == kind::BITVECTOR_UGT
-      || node.getKind() == kind::BITVECTOR_UGE
-      || node.getKind() == kind::BITVECTOR_SGT
-      || node.getKind() == kind::BITVECTOR_SGE
-      || node.getKind() == kind::BITVECTOR_ULE
-      || node.getKind() == kind::BITVECTOR_SLE
-      || node.getKind() == kind::BITVECTOR_REDOR
-      || node.getKind() == kind::BITVECTOR_REDAND
-      || (node.getKind() == kind::NOT
-          && (node[0].getKind() == kind::EQUAL
-              || node[0].getKind() == kind::BITVECTOR_ULT
-              || node[0].getKind() == kind::BITVECTOR_SLT
-              || node[0].getKind() == kind::BITVECTOR_UGT
-              || node[0].getKind() == kind::BITVECTOR_UGE
-              || node[0].getKind() == kind::BITVECTOR_SGT
-              || node[0].getKind() == kind::BITVECTOR_SGE
-              || node[0].getKind() == kind::BITVECTOR_ULE
-              || node[0].getKind() == kind::BITVECTOR_SLE
-              || node[0].getKind() == kind::BITVECTOR_REDOR
-              || node[0].getKind() == kind::BITVECTOR_REDAND)))
-  {
-    return true;
-  }
-  else
-  {
-    return false;
-  }
+  return node.getKind() == kind::EQUAL || node.getKind() == kind::BITVECTOR_ULT
```

Comment:
You can omit the `kind::NOT` part if you add `if (node.getKind() == kind::NOT) { node = node[0]; };` at the beginning.